CHANGELOG.md

```diff
@@ -17,7 +17,7 @@
 * Android Gradle Plugin 7.4.0 and above.
 
 ### Internal
-* None.
+* Updated to Realm Core 13.13.0, commit 79183be6417821431fff44a6d416a68664957c66.
```

Comment:
Don't we need to disable multiplexing? We had to do that in Kotlin because it was causing problems?

Reply:
Yes. Good catch. I did it locally to get tests to pass, but apparently stashed it at some point 😅